_data/simple-icons.json

```diff
@@ -3628,6 +3628,11 @@
             "hex": "EA4AAA",
             "source": "https://github.com/sponsors"
         },
+        {
+            "title": "gitignore",
```

Comment:
I think the name is `gitignore.io`. This is the name used everywhere on Github and in the doc.

Reply:
Ok, i change it!